jest.config.js

```diff
@@ -21,6 +21,8 @@ module.exports = {
     "/e2e-tests/",
     "/tests/",
     "/scripts/",
+    ".test_sls_next_output",
+    "__tests__",
```

Comment:
These were included in the coverage. Shouldn't be too important since they're not available in CI, but they do mess up local coverage.